### PR TITLE
apinger: fixed bugs in apinger.init and apinger.rpc

### PR DESCRIPTION
**Pull request by joed74**
fixed bug in init_apinger_config - debug/status_interval/rrd_interval were never set correctly
fixed bug in apinger_status - send and receive were swapped 
added ability top use ipv6 on wan6 (has to be fixed in luci-app-apinger as well)

Maintainer: @samm-git,  @hnyman, @jempatel
Compile tested: ARMv8 Processor rev 4, FriendlyElec NanoPi R4S, OpenWrt 22.03.3
Run tested: ARMv8 Processor rev 4, FriendlyElec NanoPi R4S, OpenWrt 22.03.3

Description: see above

### CODE REVIEW

File: net/apinger/files/apinger.rpc [hunk 1]
```diff
@@ -38,7 +38,7 @@ apinger_status() {
 		if [ -f "$status_file" ]; then
 			_IFS="$IFS"
 			IFS="|"
-			while read -r address srcip target received sent timestamp latency loss alarm; do
+			while read -r address srcip target sent received timestamp latency loss alarm; do
```
Comment (feckert) [inline net/apinger/files/apinger.rpc:41]: Is this change needed

Comment (joed74) [reply net/apinger/files/apinger.rpc:41]: According to the source, the status-format is written like this:

`fprintf(f,"%s|%s|%i|%i|%u",t->name, t->config->srcip, t->description, t->last_sent+1, t->received, t->last_received_tv.tv_sec);` - sent and then received

Without the change Sent and Received are shown incorrectly in luci. I noticed this when I only had sent packages without any received ones...

**Comment by feckert** [comment]
Could we also please bump the PKG_RELEASE?

**Comment by joed74** [comment]
Updated PKG_RELEASE from 5 to 6

**Comment by feckert** [comment]
Ok thanks. Then we have to squash this chnage to one. So wie have your change with the PKG_RELEASE bump in one commit.

Make your change in your local branch on your work machine, and then make a `git push --force`.

**Comment by joed74** [comment]
> Ok thanks. Then we have to squash this chnage to one. So wie have your change with the PKG_RELEASE bump in one commit.
> 
> Make your change in your local branch on your work machine, and then make a `git push --force`.

I tried my best, if its not ok i close and reopen a new pull request...

**Comment by feckert** [comment]
You do not need to close this pullreuqest and reopen a new one.
Just make a ` git rebase -i HEAD~5` and the squash the commits to one.
After that make a `git commit --amend` to change the commit message.
Then make a `git push --force` to your branch forked branch. 
And your pullrequest is updated. 

Please make the effort to do this that way, it will improve your skills.

**Comment by joed74** [comment]
Ok, done - thanks for the instructions!

**Comment by feckert** [comment]
Sorry but you have not a new line after the commit header.

Otherwise, the commit message does not look great. It's all in the headline.

Please rewrite the message to:
```
apinger: fixed bugs in apinger.init and apinger.rpc

Fix bug in init_apinger_config - debug/status_interval/rrd_interval were never set correctly
Fix bug in apinger_status - Send and receive are swapped
Fix bug in apinger_status - added ability to use ipv6 on wan6

```
If we want to be very precise, we would have to move the changes into separate commits, as they are different changes.

**Comment by joed74** [comment]
Changed commit message

**Comment by feckert** [comment]
Thanks for your contribution.
Merged!
